Hi3Helper.Core/Lang/en.json

```diff
@@ -127,9 +127,9 @@
     "ListCol4": "Size",
     "ListCol5": "L.CRC",
     "ListCol6": "R.CRC",
-    "Status1": "Click Check Files to run the analysis",
-    "Status2": "Loading Indexes...",
-    "Status3": "{0} file(s) have been found to be broken ({1} in total size). Click Repair Game to begin the repair process",
+    "Status1": "Click 'Check Files' to integrity validation",
```

Comment:
I think "Click 'Check Files' to verify game files integrity" feels a little bit more fitting

Reply:
Yup, that sounds good to me too, I think I made a typo there.